apache/incubator-netbeans
# [NETBEANS-1457] Fix for CSS text-align property value start is unsupported

Quick fix.  The NB CSS parser was configured for:
[ [ start | !string ]? [ start | end | left | right | center | justify | match-parent ] ] | inherit | initial

However, the most recent version of the W3C CSS3 Text Module spec (https://www.w3.org/TR/css-text-3/) states it should be:
start | end | left | right | center | justify | match-parent | justify-all | inherit | initial

Updated ide/css.editor/src/org/netbeans/modules/css/editor/module/main/properties/text.properties which fixed code completion and parsing.

Also updated Unit test for Text Module (ide/css.editor/test/unit/src/org/netbeans/modules/css/editor/module/main/TextModuleTest.java) accordingly.

To test
---------
* Create a new NB10.0 HTML or PHP project
* Create a new CSS file
* Copy & paste the following code:
.testclass1 {
    text-align: start;
}

.testclass2 {
    text-align: justify-all;
}

.testclass3 {
    text-align: start center;
}
* .testclass3 should show a parsing error, .testclass1 and .testclass2 should not (before this PR, the inverse would be true...)


Hope this helps :)
